FDK/Tools/SharedData/FDKScripts/CheckOutlinesUFO.py

```diff
@@ -12,1020 +13,1145 @@
+-noBasicChecks
+    turn off basic checks:
+    - coincident points
+    - flat curves (curves which are a straight line)
+    - colinear lines (several line segments that define
+                      the same path as a single line);
 
--setMinArea <n>  Set the minimum area for a tiny outline. Default is 25 square
-units. Subpaths with a bounding box less than this will be reported, and deleted
-if fixed.
+-setMinArea <n>
+    Set the minimum area for a tiny outline. Default is 25 square
+    units. Subpaths with a bounding box less than this will be reported, and
+    deleted if fixed.
 
--setTolerance <n>  Set the maximum deviation from a straight line allowed.
-Default is 0 design space units. This is used to test whether the control points
-for a curve define a flat curve, and whether a sequence of line segments
-defines a straight line.
+-setTolerance <n>
+    Set the maximum deviation from a straight line allowed.
+    Default is 0 design space units. This is used to test whether the control
+    points for a curve define a flat curve, and whether a sequence of line
+    segments defines a straight line.
 
--wd	write changed glyphs to default layer instead of '%s'
+-wd
+    write changed glyphs to default layer instead of '%s'
 
 -g <glyphID1>,<glyphID2>,...,<glyphIDn>
-	Check only the specified list of glyphs. The list must be
-	comma-delimited. The glyph ID's may be glyphID's, glyph names, or
-	glyph CID's. If the latter, the CID value must be prefixed with the
-	string "cid". There must be no white-space in the glyph list.
-	Examples:
-		checkOutlinesUFO -g A,B,C,69 myFont
-		checkOutlinesUFO -g cid1030,cid34,cid3455,69 myCIDFont
+    Check only the specified list of glyphs. The list must be
+    comma-delimited. The glyph ID's may be glyphID's, glyph names, or
+    glyph CID's. If the latter, the CID value must be prefixed with the
+    string "cid". There must be no white-space in the glyph list.
+    Examples:
+        checkOutlinesUFO -g A,B,C,69 myFont
+        checkOutlinesUFO -g cid1030,cid34,cid3455,69 myCIDFont
 
 -gf <file name>
-	Check only the list of glyphs contained in the specified file, The file
-	must contain a comma-delimited list of glyph identifiers. Any number of
-	space, tab, and new-line characters are permitted between glyph names
-	and commas.
+    Check only the list of glyphs contained in the specified file, The file
+    must contain a comma-delimited list of glyph identifiers. Any number of
+    space, tab, and new-line characters are permitted between glyph names
+    and commas.
 
--all Force all glyphs to be processed. This applies only to UFO fonts, where processed glyphs
-	are saved in a layer, and processing of a glyph is skipped if has already been processed.
+-all
+    Force all glyphs to be processed. This applies only to UFO fonts, where
+    processed glyphs are saved in a layer, and processing of a glyph is skipped
+    if has already been processed.
 
 """ % kProcessedGlyphsLayerName
```

Comment:
please change this to PROCD_GLYPHS_LAYER_NAME
you'll need to change the upstream imports to
```python
from ufoTools import kProcessedGlyphsLayerName as PROCD_GLYPHS_LAYER_NAME
```